dpdata/plugins/ase.py

```diff
@@ -93,13 +93,21 @@ def from_labeled_system(self, atoms: ase.Atoms, **kwargs) -> dict:
             "energies": np.array([energies]),
             "forces": np.array([forces]),
         }
-        try:
-            stress = atoms.get_stress(False)
-        except PropertyNotImplementedError:
-            pass
-        else:
-            virials = np.array([-atoms.get_volume() * stress])
+
+        # try to get virials from different sources
+        virials = atoms.info.get("virial")
+        if virials is None:
+            virials = atoms.info.get("virials")
```

Comment:
It's unclear why it has two different keys.

Reply:
It's a fallback strategy as users may use either `virial` or `virials`.

Reply:
Documentation is necessary if users are expected to do something

Reply:
I try to search in the code base but find no place to add comment for this. 

I don't think we need add extra documents for this as `dpdata` is supposed to be able to find `virial` for user automatically. My patch doesn't introduce any compatibility issue, it just make the ase plugin more robust to find `virial`.

Reply:
It finds data from non-standard keys, which needs documentation to avoid unexpected behaviors.

Reply:
I just add a comment in docstring.

Reply:
It's not rendered correctly.
![image](https://github.com/deepmodeling/dpdata/assets/9496702/b6a77baa-3659-4ab9-a253-364fbc5bfb07)